examples/schelling.jl

```diff
@@ -327,3 +327,86 @@ adf
 
 # We nicely see that the larger `:min_to_be_happy` is, the slower the convergence to
 # "total happiness".
+
+# ## Saving the model state
+
+# It is often useful to save a model after running it, so that multiple branching
+# scenarios can be simulated from that point onwards. For example, once most of
+# the population is happy, let's see what happens if some more agents occupy the
+# empty cells. The new agents could all be of one group, or belong to a third, new, group.
+# Simulating this needs multiple copies of the model. `Agents.jl` provides the
+# functions [`AgentsIO.save_checkpoint`](@ref) and [`AgentsIO.load_checkpoint`](@ref)
+# to save and load models to JLD2 files respectively.
+
+# First, let's create a model with 200 agents and run it for 40 iterations.
+
+model = initialize(numagents = 200, min_to_be_happy = 5, seed = 42)
+run!(model, agent_step!, 40)
+
+figure, _ = abm_plot(model; ac = groupcolor, am = groupmarker, as = 10)
+figure
+
+# Most of the agents have settled happily. Now, let's save the model.
+
+AgentsIO.save_checkpoint("schelling.jld2", model)
+
+# Note that we can now leave the REPL, and come back later to run the model,
+# right from where we left off.
+
+model = AgentsIO.load_checkpoint("schelling.jld2"; scheduler = Schedulers.randomly)

```

Comment:
The way I'd fix this documenter failed load for the moment would be like this:

```julia
# Note that we can now leave the REPL, and come back later to run the model,
# right from where we left off.
#
# ```julia
# model = AgentsIO.load_checkpoint("schelling.jld2"; scheduler = Schedulers.randomly)
# ```
model = AgentsIO.load_checkpoint("../schelling.jld2"; scheduler = Schedulers.randomly) # hide
```